apps/ledger-live-desktop/src/renderer/screens/manager/AppsList/Item.tsx

```diff
@@ -86,7 +86,7 @@ const Item = ({
 
   const flag = currencyId
     ? Object.entries(currencyFlags || {}).find(([flagName]) => {
-        return flagName.includes(currencyId);
+        return flagName.replace("feature_currency_", "") === currencyId;
```

Comment:
Then you might as well directly use
```ts
useFeature(camelCase(`currency_${currencyId}`))
```
I have a suspicion that it might fix [your other issue](https://ledgerhq.atlassian.net/browse/LIVE-8489).

The problem might be with `getAllFeatures`, if you check how it's defined in `FirebaseRemoteConfigProvider` it captures the value of `remoteConfig`, which is only set once at app start, so `getAllFeatures` might be returning only the values that the feature flags had at that first load.
It also completely ignores the local overrides.

Reply:
If this is right we should either get rid of getAllFeatures or fix in both apps it so that it is aligned with the way the rest of the feature flags work in LL (support for overrides and also use the same flag name formatting without feature_ and in camelCase).

Reply:
i'm not exactly sure if we can go with @ofreyssinet-ledger 's suggestion, but the current change is indeed no super performant (the current approach to have to "find" through entries isn't performant already, and it makes it worse. and we do this for each item of the manager list without any caching :o

Reply:
@ofreyssinet-ledger Thanks for the suggestion, I used it and it works. However I don't want to refactor `getAllFlags` in this PR, neither create a full mapping between `FeatureId` and `Currency.id` because this will be reworked properly when we work on our Coin Config feature.